py3tftp/tftp_parsing.py

```diff
@@ -79,3 +79,18 @@ def timeout_parser(val, lower_bound=1, upper_bound=255):
              '({1} - {2})').format(value, lower_bound, upper_bound))
     else:
         return value
+
+def windowsize_parser(val, lower_bound=1, upper_bound=65535):
+    """
+    Parses and validates the 'windowsize' option against the RFC 7440.
+    """
+    value = int(val)
+    if value > upper_bound:
+         return value - (value - upper_bound)
```

Comment:
I think there's an extra space (or 2?) in front of `return` - it doesn't align with eg. the other `return` or `raise` statements. Could you align the return?